x-pack/plugins/apm/server/feature.ts

```diff
@@ -20,19 +21,15 @@ export const APM_FEATURE = {
   privileges: {
     all: {
       app: ['apm', 'kibana'],
-      api: [
-        'apm',
-        'apm_write',
-        'actions-read',
-        'actions-all',
-        'alerting-read',
-        'alerting-all',
-      ],
+      api: ['apm', 'apm_write', 'actions-read', 'actions-all'],
       catalogue: ['apm'],
       savedObject: {
         all: ['alert', 'action', 'action_task_params'],
```

Comment:
Not only shouldn't they need it... it shouldn't work, as it's a hidden type....
I guess it ignores it if its hidden and just explodes if they tried to use it.

At the time I left it because it's APM's code, but I can remove it as they probably just added it because they thought they needed to... we now no longer allow it anyway so... yeah 🤷